### PR TITLE
Fixing Documentation issues with Secrets Manager

### PR DESCRIPTION
**Pull request by elamaran11**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted. Consult the CONTRIBUTING guide for submitting pull-requests.

### More

- [X] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [X] Yes, I ran `pre-commit run -a` with this PR
- [X] Yes, I have added a new example under [examples](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/examples) to support my PR (when applicable)
- [X] Yes, I have updated the [Pages](https://github.com/aws-observability/terraform-aws-observability-accelerator/tree/main/docs) for this feature

**Note**: Not all the PRs required examples and docs.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: modules/eks-monitoring/add-ons/external-secrets/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # External Secrets Operator Kubernetes addon
 
-This deploys an EKS Cluster with the External Secrets Operator. The cluster is populated with a ClusterSecretStore and ExternalSecret using Grafana API Key secret from AWS Secret Manager. A secret store for each AWS Secret Manager is created. Store use IRSA (IAM Roles For Service Account) to retrieve the secret values from AWS.
+This deploys an EKS Cluster with the External Secrets Operator. The cluster is populated with a ClusterSecretStore and ExternalSecret using Grafana API Key secret from AWS SSM Paramete Store. A secret store for each AWS SSM Paramete Store is created. Store use IRSA (IAM Roles For Service Account) to retrieve the secret values from AWS.
```
Comment (bonclay7) [inline modules/eks-monitoring/add-ons/external-secrets/README.md:3]: typo on Parameter store

File: docs/eks/index.md [hunk 2]
```diff
@@ -93,7 +93,7 @@ Ensure you have necessary IAM permissions (`CreateWorkspaceApiKey, DeleteWorkspa
 
 !!! note
     Starting version v2.5.x and above, we use Grafana Operator and External Secrets to
-    manage Grafana contents. Your API Key will be stored securely on AWS Secrets Manager
+    manage Grafana contents. Your API Key will be stored securely on AWS SSM Paramter Store
```
Comment (bonclay7) [inline docs/eks/index.md:96]: typo

File: docs/contributors.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ Our goal is to make it easier to use AWS Open Source Observability Services.
 The core team include the following people:
 
 * Abhi Khanna
+* Elamaran Shanmugam
```
Comment (bonclay7) [inline docs/contributors.md:12]: can you remove Ramesh please?

**Review by bonclay7 — COMMENTED** [review]
Awesome, just a few typos

File: modules/eks-monitoring/add-ons/external-secrets/README.md (hunk 1 above)
Comment (elamaran11) [reply modules/eks-monitoring/add-ons/external-secrets/README.md]: Fixed

File: docs/eks/index.md (hunk 2 above)
Comment (elamaran11) [reply docs/eks/index.md]: Fixed

**Comment by elamaran11** [comment]
> Awesome, just a few typos

@bonclay7 Fixed all issues, Please review and merge.

**Review by bonclay7 — APPROVED** [review]
Awesome, thanks!

**Pull request merged**